src/plots/plots.js

```diff
@@ -132,32 +132,12 @@ plots.registerSubplot = function(_module) {
     var plotType = _module.name;
 
     if(subplotsRegistry[plotType]) {
-        throw new Error('plot type' + plotType + ' already registered');
-    }
-
-    var attr = _module.attr,
-        idRoot = _module.idRoot;
-
-    var regexStart = '^',
-        regexEnd = '([2-9]|[1-9][0-9]+)?$',
-        hasXY = (plotType === 'cartesian' || subplotsRegistry === 'gl2d');
-
-    function makeRegex(mid) {
-        return new RegExp(regexStart + mid + regexEnd);
+        console.log('plot type ' + plotType + ' already registered');
```

Comment:
Ooops the above comment (https://github.com/plotly/plotly.js/pull/193/files#r50156440) was supposed to be placed here.